### ISSUE TITLE
Inconsistency in argument names across BlanketCutter shapes

### ISSUE THREAD

**Issue by RemDelaporteMathurin** [ukaea/paramak#578]
`BlanketCutterParallel `has a `thickness ` argument, whereas `BlanketCutterStar `has a `distance `argument. 

What should we do ? @Shimwell @billingsley-john

**Comment by Shimwell** [comment]
I vote distance in both cases as this is the argument name in the extrude shapes.

**Comment by RemDelaporteMathurin** [comment]
True, but the user could be confused as there are many distances:
![image](https://user-images.githubusercontent.com/40028739/100252314-0304f100-2f40-11eb-89de-934f8f4ad96a.png)

Distance is also suitable for width or height. In ExtrudeShape, there is not doubt cause there is only one extrusion distance possible.

**Comment by billingsley-john** [comment]
I think that whatever word is used, whether that be 'width', 'length', 'thickness', 'distance', there will be some ambiguity. Like @Shimwell, I like the use of 'distance' for consistency with the extrude shapes as that parameter is the distance extruded by the function

**Comment by RemDelaporteMathurin** [comment]
Ok then. The only potential issue I find is that the `distance `parameter from the Cutter class and from the inherited class don't have the same value. 

This may mess with the setters /getters being overriden
